build-system/tasks/extension-generator/jscodeshift/insert-extension-bundles-config.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const arrayAssignmentIdName = 'extensionBundles';
+
+/**
+ * Inserts an extension entry into bundles.config.js
+ *
+ * ```
+ *    const extensionBundles = [
+ *      ...,
+ *   +  {
+ *   +    name: "amp-foo",
+ *   +    version: "1.0",
+ *   +  }
+ *      ...,
+ *    ]
+ * ```
+ *
+ * Properties of inserted object are defined by --insertExtensionBundle:
+ *  - name
+ *  - version
+ *  - latestVersion (optional)
+ *
+ * @param {*} file
+ * @param {*} api
+ * @param {*} options
+ * @return {*}
+ */
+export default function transform(file, api, options) {
+  const j = api.jscodeshift;
+
+  const {insertExtensionBundle} = options;
+
+  if (!insertExtensionBundle) {
+    throw new Error('No insertExtensionBundle option');
+  }
+
+  const {name, version, latestVersion} = insertExtensionBundle;
```

Comment:
Do you have to `JSON.parse`? It's a json string, right?

Reply:
`jscodeshift` parses it by default.